src/org/labkey/test/components/ui/navigation/apps/LKAppsMenu.java

```diff
@@ -0,0 +1,69 @@
+package org.labkey.test.components.ui.navigation.apps;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.html.BaseBootstrapMenu;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Wraps the expand/collapse toggle to show/hide the Apps-navigation menu in LabKey pages
+ * In SM or Biologics views, AppsMenu is the analog of this component
+ */
+public class LKAppsMenu extends BaseBootstrapMenu
```

Comment:
It kind of feels like this could be merged with AppsMenu. If I understand correctly this is the "button" on the title bar, maybe if the locators in the getToggleLocator and in the finder used an 'or' condition you could combine these two files (`Locator.xpath("(//li[@id='headerProductDropdown']/a) | (//button[@id='product-navigation-button'])")`).

Maybe I'm missing something.

Reply:
This is a good suggestion, I just worried that if I tried to get too fancy with those two parts you guys would ask me to separate them.
I'm willing to try to unify these two into one that can straddle both worlds, tho